src/main/java/de/tum/in/www1/artemis/domain/Feedback.java

```diff
@@ -70,6 +70,7 @@
     @ManyToOne
     private GradingInstruction gradingInstruction;
 
+    // TODO: JP remove these two references as they are not really needed
```

Comment:
Actually, those references ensure the deletion of referenced conflicts when the feedback is deleted.

Reply:
We now, but we don't like the implementation, so we want to find a better way in the future